staging/src/k8s.io/kubectl/pkg/cmd/cp/cp.go

```diff
@@ -252,9 +252,11 @@ func (o *CopyOptions) Run() error {
 	}
 
 	if len(srcSpec.PodName) != 0 {
+		cmdutil.CheckErr(o.checkContainerCommandExist("tar", srcSpec))
```

Comment:
Is there a reason you're forcibly exiting (which is what CheckErr does) instead of returning the error?

Nit: I would slightly prefer these checks to be done from within copy functions, since the fact that those use tar is an implementation detail of each.

Reply:
> Is there a reason you're forcibly exiting (which is what CheckErr does) instead of returning the error?
> 
> Nit: I would slightly prefer these checks to be done from within copy functions, since the fact that those use tar is an implementation detail of each.

I agree with you，I will move these checks within copy function。